quix-frontend/client/src/lib/runner/services/autocomplete/highlight-and-score.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  createCaption,
+  createMatchMask,
+  findAllIndexOf,
+} from './autocomplete-utils';
+import { findRelevantPartOfPrefix } from "../../../sql-autocomplete/adapter/sql-autocomplete-adapter-utills";
+import { ICompleterItem as AceCompletion } from '../../../code-editor/services/code-editor-completer';
+import { ContextType, QueryContext } from '../../../sql-autocomplete/sql-context-evaluator';
+
+const PERFECT_SCORE = 10000;
+
+export function highlightAndScore(
+  autocompletionSuggestions,
+  queryContext,
+  searchInObject
+) {
+  const lowerCasedPrefix = queryContext.prefix.trim().toLowerCase();
+  const isInObjectContext = queryContext.contextType !== ContextType.Undefined && searchInObject;
```

Comment:
no need to check if it is defined context. it will asways be defined if search in object true

Reply:
fixed! 

```
return searchInObject
    ? enrichCompletionForObject(queryContext.prefix, autocompletionSuggestions, queryContext, lowerCasedPrefix)
    : enrichCompletionItem(autocompletionSuggestions, lowerCasedPrefix);